src/tmp_dir.cpp

```diff
@@ -57,7 +57,11 @@ TmpDir::TmpDir(std::string prefix)
 
 void TmpDir::Execute(std::string exe, std::string args)
 {
-    std::string cd  = "cd " + this->path.string() + "; ";
+    if(miopen::IsEnabled(MIOPEN_DEBUG_SAVE_TEMP_DIR{}))
```

Comment:
What is the purpose of this env?

If it is to set tmp directory to anything other than /tmp, then, please rename this variable to MIOPEN_DEBUG_TMP_DIR_PATH

Or if it is to simply dump tmp directory path into log, why we need an env variable. Just raise the log level to the highest level and it will be dumped.

Reply:
I didn't add this env. It was already there. It's not /tmp. It's going to be something like 

/tmp/miopen-YOUR_KERNEL_NAME-XXXXX

We need to know the exact path in order to know where we can check the ISA. HIP-clang doesn't allow you to specify where to dump the isa

Reply:
I see, If the env was already there and you are simply using it, I hope, the dumping full path of isa is its intended purpose, because the name appeared confusing.

Reply:
sorry, line #60 is added by you. Is that not right? Can we not simply dump without using this variable?

Reply:
MIOPEN_DEBUG_SAVE_TEMP_DIR is to save tmp_dir
MIOPEN_DEBUG_HIP_DUMP is to dump ISA. Yes, it will be saved to tmp_dir. 

So if you want check ISA,,you would need to set both of them to be true. But these are 2 separate things. I'm not sure it's a good idea to mix them